code/game/machinery/computer/arcade.dm

```diff
@@ -614,7 +614,7 @@
 						L.Stun(200, ignore_canstun = TRUE) //you can't run :^)
 					var/S = new /obj/singularity/academy(usr.loc)
 					addtimer(CALLBACK(src, /atom/movable/proc/say, "[S] winks out, just as suddenly as it appeared."), 50)
```

Comment:
It seems like this should call on the singulo as well

Reply:
I think the text is supposed to come from the arcade machine.

Reply:
No, it spawns it where the player is.
It's the academy singuloth which doesn't move or grow.